backend/proxy-server/README.md

```diff
@@ -52,5 +52,15 @@ to have as a pre-requisite to running the test suite.
   - all field names are case sensitive for the purpose of testing
   - the fields **should** be of type `Single line text`
 
-[pre-defined pseudo-random IPs]: ../lib/fake-ips.js
+[pre-defined pseudo-random ips]: ../lib/fake-ips.js
 [script]: ../util/loopback-network.sh
+
+### Running tests
+
+To execute the gateway tests, execute the following commands in the order after adding the necessary configs in .env.conduit-user file.
+
+1. Add the proxys - `./util/loopback-network.sh`
```

Comment:
this should be `./util/loopback-network.sh add`.

The script takes one argument which is either of `add` or `del`.